core/src/main/java/com/google/zxing/datamatrix/encoder/HighLevelEncoder.java

```diff
@@ -137,15 +137,18 @@ public static String encodeHighLevel(String msg) {
    *                {@code SymbolShapeHint.FORCE_SQUARE} or {@code SymbolShapeHint.FORCE_RECTANGLE}.
    * @param minSize the minimum symbol size constraint or null for no constraint
    * @param maxSize the maximum symbol size constraint or null for no constraint
+   * @param forceC40 enforce C40 encoding
    * @return the encoded message (the char values range from 0 to 255)
    */
   public static String encodeHighLevel(String msg,
```

Comment:
Let's keep an overload with the original signature, to avoid breaking the public API